tutorial/07_random_inputs/run.py

```diff
@@ -0,0 +1,20 @@
+#Dummy example of ipython script to evaluate student answer for equation: 2x²-Value=0
+import math
+
+#Get the random input parameter and select a predifined value based on it
+RandomIndex=int(float(get_input("@random")[0]*100))%6
+Values = [8, 18, 32, 50, 72, 98]
+Value=Values[RandomIndex]
+
+#Get the student answer and the correct answer
+Student = int(get_input("student_code"))
+Solution = int(math.sqrt(Value/2))
+
+#As an example, the two answers can be accessed from command line via temporary environment variables
+#This is useful if you want to share the same variable for python commands and bash commands
+%env MySolution=$Solution
```

Comment:
Same remarks for Python naming convention: variable names in lower case. As we are mixing shell script, the environment variables can be in uppercase.